src/test/kotlin/chapter7/exercises/ex10/listing.kt

```diff
@@ -5,11 +5,6 @@ import java.util.concurrent.Future
 
 typealias Par<A> = (ExecutorService) -> Future<A>
 
-object Listing {
-    //tag::init[]
-    fun <K, V> choiceMap(
-        key: Par<K>,
-        choices: Map<K, Par<V>>
-    ): Par<V> = TODO()
-    //end::init[]
-}
+fun <A> choiceN(n: Par<Int>, choices: List<Par<A>>): Par<A> = TODO()
```

Comment:
Did you leave out the asciidoc tag markers on purpose?

Reply:
All exercise numbers from 7.10 on were incremented by one because an exercise was misclassified as an example and was numbered outside the exercise series. This is why there seem to be so many differences between former and new exercises 7.10, former and new 7.11, and so on.